index.js

```diff
@@ -4,13 +4,14 @@
 var express = require('express');
 var ParseServer = require('parse-server').ParseServer;
 
-var databaseUri = process.env.DATABASE_URI || process.env.MONGOLAB_URI
+var databaseUri = process.env.DATABASE_URI || process.env.MONGOLAB_URI;
 
 if (!databaseUri) {
   console.log('DATABASE_URI not specified, falling back to localhost.');
 }
 
 var api = new ParseServer({
+  serverURL: process.env.PARSE_SERVER_URL,
```

Comment:
https://github.com/ParsePlatform/parse-server-example/pull/75/files already added this as SERVER_URL. I think we have dups now: https://github.com/ParsePlatform/parse-server-example/blob/master/index.js